docs/workflows/api-token.md

```diff
@@ -546,3 +546,48 @@ In order to validate a token offline, the JWT header and claims must be decoded.
 4. Check timestamps in JWT claims where `nbf` <= current time as unix timestamp in seconds <= `exp`
 5. Check that JWT claims target tenant `bi_t` and target realm `bi_r` match the tenant and realm for the given application.
 6. Check that JWT claims has the expected scopes.
+
+## Listing access tokens
+
+Our API also provides the functionality to list active tokens that are held by a given identity or an application.
+In order to list tokens, you need to know the ID of the application that was used to issue the token.
+
+You need to use a Management API Bearer token with the `tokens:read` scope in order to call this API.
+
+### Request: Listing access tokens for an identity
+
+```bash
```

Comment:
I was using https://curlconverter.com to create the other language code snippets from curl and then wrapping them in the <Tabs>. I can help with this part if you want or need.

Reply:
I just updated this file with a `MultiLanguageCodeBlock` to handles this. If you merge main you should see how to add the curl to this block.

Reply:
Thank you so much, this is excellent!